homeassistant/components/person/__init__.py

```diff
@@ -50,7 +50,7 @@
 })
 
 CONFIG_SCHEMA = vol.Schema({
-    vol.Optional(DOMAIN): vol.Any(vol.All(cv.ensure_list, [PERSON_SCHEMA]), {})
+    vol.Optional(DOMAIN): vol.All(cv.ensure_list, vol.Any([PERSON_SCHEMA], []))
```

Comment:
Would it work to accept `[{}]` in the schema? Then we wouldn't need to change `cv.ensure_list`.

Reply:
I guess it could, but will have to put in a specific check in person's setup to ignore empty lists, likely [here](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/person/__init__.py#L83): `if not conf: continue`.

Reply:
Right. You mean empty dict, right? Yeah, that's also inconvenient.

Reply:
Empty dict yes. Guess it's better than fiddling with `ensure_list`, which is used everywhere